### ISSUE TITLE
Speech Recognition in German?

### ISSUE THREAD

**Issue by Fechulo** [zlsa/atc#338]
For some reason, when I enable speech recognition in chrome, the voice is in german. This means that numbers are said in german (e.g., Delta eins zwei drei) and sometimes weird things happen in EDDM where the fixes that start with "DM" are pronounced "Deutsche Mark 1 2 3". None of my settings anywhere include german and I can't find a way to fix it.
Anyone have any ideas?
Thanks!

**Comment by erikquinn** [comment]
I looked into this, but couldn't find where the voice was being specified, so I don't know if there's an English one that can be used instead. there's also issues with how the numbers are said and other phraseology inconsistencies that I am in the process of addressing. You can have a look around in my "phraseology" branch, and it should be apparent how I addressed it.

**Comment by Fechulo** [comment]
I'm pretty sure it has nothing to do with the game itself, I'd say it's something Chrome's messing up somehow. I just wanted to know if you knew why this is happening and how to solve it.

**Comment by harp71** [comment]
"Deutsche Mark 1 2 3" ... ROFL

**Comment by erikquinn** [comment]
GOT IT!

[https://dvcs.w3.org/hg/speech-api/raw-file/tip/speechapi.html#utterance-attributes](https://dvcs.w3.org/hg/speech-api/raw-file/tip/speechapi.html#utterance-attributes)

```
var utterance = new SpeechSynthesisUtterance(textToSay); //make the object
utterance.lang = "en-US"; //set the language!!
prop.speech.synthesis.speak(utterance);  //say the words
```

Replacing the `prop.speech.synthesis.speak(new SpeechSynthesisUtterance(textToSay));` with the above seems to work! It's an american lady voice then, and the documentation seems to say there's a bunch of voices for every language, so with a little reading we might be able to get it talking correctly, AND maybe even associating particular voices with particular aircraft, which is gimmicky, but neat!

**Comment by erikquinn** [comment]
This is resolved since #365 has been merged. Recommend it be closed.

**Comment by engeplabs** [comment]
Still not closed yet...
